clean_air/data/data_store_access.ipynb

```diff
@@ -0,0 +1,354 @@
+      "Name: WGS 84\n",
+      "Axis Info [ellipsoidal]:\n",
+      "- Lat[north]: Geodetic latitude (degree)\n",
+      "- Lon[east]: Geodetic longitude (degree)\n",
+      "Area of Use:\n",
+      "- name: World.\n",
+      "- bounds: (-180.0, -90.0, 180.0, 90.0)\n",
+      "Datum: World Geodetic System 1984 ensemble\n",
+      "- Ellipsoid: WGS 84\n",
+      "- Prime Meridian: Greenwich\n",
+      ", description='This is a test data set', data_type=<DataType.OTHER: 'other'>, contacts=[])\n"
+     ]
+    }
+   ],
+   "source": [
+    "test_dataset_id = \"testdataset\"\n",
+    "\n",
+    "dataset = dataset_store.get(test_dataset_id)\n",
+    "print(str(dataset))\n",
+    "\n",
+    "metadata = metadata_store.get(test_dataset_id)\n",
+    "print(str(metadata))"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "source": [
+    "## Deployment\n",
+    "For read-only/anonmous access to work correctly, the data storage bucket must have this policy applied:\n",
+    "```json\n",
+    "{\"Version\": \"2008-10-17\",\n",
+    " \"Id\": \"Read Access For Anonymous Users\",\n",
+    " \"Statement\": [\n",
+    "    {\n",
+    "      \"Sid\": \"Read-only and list bucket access for Everyone\",\n",
+    "      \"Effect\": \"Allow\",\n",
+    "      \"Principal\": {\"anonymous\": [\"*\"]},\n",
+    "      \"Resource\": \"*\",\n",
+    "      \"Action\": [\"GetObject\", \"ListBucket\"]\n",
+    "    }\n",
+    "]\n",
+    "}\n",
+    "```\n",
+    "Note, this policy was written for and tested with the JASMIN object store, so may need tweaking for use with\n",
+    "other object stores."
+   ],
+   "metadata": {
+    "collapsed": false,
+    "pycharm": {
+     "name": "#%% md\n"
+    }
+   }
+  },
+  {
+   "cell_type": "markdown",
+   "source": [
+    "## Developer Note: s3fs vs s3fs-fuse\n",
+    "\n",
+    "[s3fs](https://pypi.org/project/s3fs/) is a python library that provides a filesystem-like interface,\n",
+    "but actually just uses the S3 APIs under the hood and doesn't interact with the local filesystem\n",
+    "\n",
+    "[s3fs-fuse](https://github.com/s3fs-fuse/s3fs-fuse) actually mounts an S3 compatible bucket as a virtual filesystem,\n",
+    "which can be accessed using any method that works for local files."
+   ],
+   "metadata": {
+    "collapsed": false
+   }
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "outputs": [],
+   "source": [],
+   "metadata": {
+    "collapsed": false,
+    "pycharm": {
+     "name": "#%%\n"
+    }
+   }
+  }
+ ],
+ "metadata": {
+  "kernelspec": {
+   "display_name": "Python 3 (ipykernel)",
+   "language": "python",
+   "name": "python3"
+  },
+  "language_info": {
+   "codemirror_mode": {
+    "name": "ipython",
+    "version": 3
+   },
+   "file_extension": ".py",
+   "mimetype": "text/x-python",
+   "name": "python",
+   "nbconvert_exporter": "python",
+   "pygments_lexer": "ipython3",
+   "version": "3.8.12"
+  }
+ },
+ "nbformat": 4,
+ "nbformat_minor": 1
+}
```

Comment:
Beautiful.  I will have to look at the actual notebooks at some point as well as the source code, but I love all the information you have put in here and the way it is organized.  Looks like it makes it easy for beginners and experts alike.